src/ApparentHorizons/StrahlkorperFunctions.hpp

```diff
@@ -39,4 +151,219 @@ template <typename Fr>
 void radius(const gsl::not_null<Scalar<DataVector>*> result,
             const Strahlkorper<Fr>& strahlkorper);
 /// @}
+
+/// @{
+/*!
+ * `cartesian_coords(i)` is \f$x_{\rm surf}^i\f$, the vector of \f$(x,y,z)\f$
+ * coordinates of each point on the Strahlkorper surface.
+ *
+ * `radius` is the radius as a function of angle, as returned by
+ * `StrahlkorperFunctions::radius`, and `r_hat` is the Euclidean
+ * radial unit vector as returned by `StrahlkorperFunctions::rhat`.
```

Comment:
(optional) I suggest to structure these docs with Doxygen commands:

```
 * \param radius The radius as a function of angle, as returned by ...
 * \param r_hat The Euclidean radial unit vector ...
```

This would apply to all functions in this namespace, if you choose to do it.